src/modules/AdvancedPaste/AdvancedPaste/Helpers/JsonHelper.cs

```diff
@@ -33,6 +34,19 @@ internal static class JsonHelper
         private static readonly Regex CsvRemoveStartAndEndQuotationMarksRegex = new Regex(@"^""(?=(""{2})+)|(?<=(""{2})+)""$");
         private static readonly Regex CsvReplaceDoubleQuotationMarksRegex = new Regex(@"""{2}");
 
+        private static bool IsJson(string text)
+        {
+            try
+            {
+                using JsonDocument doc = JsonDocument.Parse(text);
```

Comment:
Is this really auto disposed? Not better to use underscore?


```suggestion
                _ = JsonDocument.Parse(text);
```